perf/blogbench.py

```diff
@@ -0,0 +1,62 @@
+import os
+from avocado import Test
+from avocado import main
+from avocado.utils import process
+from avocado.utils import build
+from avocado.utils import archive
+from avocado.utils.software_manager import SoftwareManager
+from avocado.core import data_dir
+
+
+class Blogbench(Test):
+
+    ''' Blogbench will start the required threads and the test will run
+        according to the args given.A final "score" will then be given as
+        an indication of read and write performance '''
+
+    def setUp(self):
+        sm = SoftwareManager()
+        # Check for basic utilities
+        deps = ['gcc', 'make']
+        for package in deps:
+            if not sm.check_installed(package) and not sm.install(package):
+                self.error(package + ' is needed for the test to be run')
+        url = 'https://download.pureftpd.org/blogbench/blogbench-1.1.tar.bz2'
+        blogbench_url = self.params.get('blogbench_url',
+                                        default=url)
+        blogbench_tarball = self.fetch_asset(blogbench_url, expire='7d')
+        archive.extract(blogbench_tarball, self.srcdir)
+        blogbench_version = os.path.basename(blogbench_tarball
+                                             .split('.tar.')[0])
+        self.blogbench_dir = os.path.join(self.srcdir, blogbench_version)
+        os.chdir(self.blogbench_dir)
+        patch = self.params.get('patch', default='config_guess.patch')
+        process.run('patch -p1 config.guess %s' % self.datadir +
+                    '/' + patch, shell=True)
+        process.system('./configure')
+        build.make(self.blogbench_dir, extra_args='install-strip')
+
+    def test(self):
+        test_dir = self.params.get('test_dir', default=data_dir.get_tmp_dir())
+        # 4 Different types of threads can be specified as an args
+        # These args are given higher value to stress the system more
+        # Here, test is run with default args
+        args = self.params.get('args', default='')
+        args = ' -d ' + test_dir + ' ' + args
```

Comment:
string add using + not looks good can you please do this way 

` args = ' -d %s  %s'  % ( test_dir, args)`